src/color.js

```diff
@@ -516,7 +572,7 @@ export default class Color {
 			return XYZ;
 		}
 
-		let env = {W1, W2, XYZ, options};
+		let env = {W1, W2, XYZ, ...options};
```

Comment:
This was intentional. Now plugins cannot get the originally provided options doing `env.options` and cannot branch accordingly. Also, any existing references to `env.options` will break.

This is not a biggie for this particular case (I doubt there are that many plugins using `env.options`, though I think there are some official ones), but we do use `env.options` across the codebase, and it would break that convention.

Reply:
Oh! I guess I will only keep the JSDoc changes then. Cheers.

Reply:
Thanks! Ping me when this is ready to merge!

Reply:
I have reworked the injection of `env.M` to be explicit from `options`. I understand that adding a new param would be more consistent with the other place where `env` is used, but putting in another positional parameter itself is a compatibility break I do not wish to go for.

Reply:
Why do you need `M` on `env`?